compose/compose-file/build.md

```diff
@@ -299,6 +310,123 @@ build:
   target: prod
 ```
 
+### secrets
+`secrets` grants access to sensitive data defined by [secrets](index.md#secrets) on a per-service build basis. Two
+different syntax variants are supported: the short syntax and the long syntax.
+
+Compose implementations MUST report an error if the secret isn't defined in the
+[`secrets`](index.md#secrets-top-level-element) section of this Compose file.
```

Comment:
? 
Unless the context is a sample. It didn't seem to be the case.
```suggestion
[`secrets`](index.md#secrets-top-level-element) section of a Compose file.
```

Reply:
Maybe "of the Compose file"?

What it's trying to say is that if you reference a secret that you haven't defined, an error should be returned.